Library/Homebrew/formula_installer.rb

```diff
@@ -398,7 +433,12 @@ def finish
 
     keg = Keg.new(formula.prefix)
     link(keg)
-    fix_install_names(keg) if OS.mac?
+
+    if OS.mac?
+      # this needs to be changed to a test against build_bottle? and
+      # formula.bottle.needs_relocation?
+      fix_install_names(keg) unless formula.name == 'cctools'
```

Comment:
Adjusted! I also had to add a new guard around the new `keg.relocate_install_names` call in `FormulaInstaller#pour` with similar behavior.